### ISSUE TITLE
Non-unique IDs on the homepage

### ISSUE THREAD

**Issue by sapegin** [survivejs/site#102]
![image 2018-06-22 at 7 37 01 pm](https://user-images.githubusercontent.com/70067/41790699-a9942322-7653-11e8-8dd6-82e305a88c0d.png)

Can we disable links on heading on the main page?

**Comment by bebraw** [comment]
I have a better solution [here](https://github.com/ReactFinland/site/tree/master/components). `AnchorProvider` and `AnchorHeader` work in pair to solve it. To integrate, it might need the same Markdown solution too.

**Comment by sapegin** [comment]
Yup, if we want to. But I think it's unnecessary on marketing pages.
